guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,46 @@
+[id="migrating-to-a-new-server}]_{context}"]
```

Comment:
Fix the syntax for the IDs, also all others
In this case, it should be: `[id="migrating-to-a-new-server_{context}"]`